doc/man-pages/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ IF (PROG_GZIP AND PROG_POD2MAN)
 		add_dependencies(man_pages_all man_page_${appname})
 		if (UNIX)
 			INSTALL(FILES "${MRPT_BINARY_DIR}/man-pages/${appname}.1.gz" DESTINATION ${mrpt_apps_INSTALL_PREFIX}share/man/man1/ )
-		endif()
+		endif(UNIX)
```

Comment:
No need to change this, but just FYI: the verbose CMake notation of `if(XXX) ... endif(XXX)` is sort of a relic from very old CMake versions, so new CMake code shouldn't have it. In fact, for MRPT 2.0.0 one of the pending tasks will be make a clean up of those redundant expressions ;-)